spec/serializer_spec.rb

```diff
@@ -826,7 +832,7 @@ def read_attribute_for_validation(attr)
         ],
       }
       # Also test that it handles string include arguments.
-      includes = 'long-comments, long-comments.post.author'
```

Comment:
This changes the behavior of the test, it's specifically testing the behavior of "overlapping include paths" but this no longer does that.

Reply:
I fixed it, but it still shows like it's not 🤔